CMakeLists.txt

```diff
@@ -94,3 +98,23 @@ if(QHOTKEY_INSTALL)
 
     export(TARGETS qhotkey FILE QHotkeyConfig.cmake)
 endif()
+
+if(UNIX)
+    # General settings
+    set(COMPRESION_TYPE "xz")
```

Comment:
Let's don't hardcode the compression type too.

Reply:
Line removed.